MiaWrapper.py

```diff
@@ -37,7 +47,7 @@ def sortImages(images, modalities):
                 currentImageLocation = modalityImage['location']
                 filename = os.path.basename(currentImageLocation)
                 newImageLocation = os.path.join(newImageDirectory, filename)
-                os.rename(currentImageLocation, newImageLocation)
+                copyfile(currentImageLocation, newImageLocation)
```

Comment:
Apparently os.symlink is not available in windows:
> Availability: Unix.
https://docs.python.org/2/library/os.html#os.symlink